src/pages/index.tsx

```diff
@@ -16,8 +30,73 @@ export const getServerSideProps = async () => {
   };
 };
 
-const Home = ({ todos }: Props) => {
-  return <TodoTable todos={todos} />;
+const TodoTable: ({ todos }: { todos: ReceiveTodo[] }) => JSX.Element = ({
+  todos,
+}) => {
+  const [Todos, setTodos] = useState(todos);
+  const { isLoggedIn } = useUserInfo();
+
+  const snackbar = useSnackbar();
+
+  const { mutate } = useMutation(deleteTodo);
+
+  const handleDelete = (id) => {
+    Todos.splice(id, 1);
```

Comment:
This seems wrong (`splice(start, deleteCount)`). First element should not be id but array position